code/modules/hydroponics/plant_genes.dm

```diff
@@ -203,13 +232,30 @@
 	for(var/datum/plant_gene/trait/T in G.seed.genes)
 		T.on_slip(G, M)
 
+/datum/plant_gene/trait/slip/pod_Crossed(datum/species/pod/P,mob/living/carbon/human/H,atom/movable/AM)
```

Comment:
don't do this, there is a slippery component. add one to your appropriate mob when you get the slip trait

https://github.com/tgstation/tgstation/blob/master/code/datums/components/slippery.dm#L6